### PR TITLE
replaced link to GitHub repository

### PR DESCRIPTION
**Pull request by BeateRixen**
All links to our GitHub repos currently still point to the Software AG address: github.com/SoftwareAG. We need to change this to point to the new Cumulocity IoT Github address. Please approve for your areas.

### CODE REVIEW

File: content/protocol-integration/lpwan-custom-codec-bundle/deploying-sample-codec-microservice.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ layout: redirect
 
 Steps to build the example codec `lora-codec-lansitec` microservice.
 
-1. Clone the https://github.com/SoftwareAG/cumulocity-examples.git repository.
+1. Clone the {{< link-c8y-github >}}/cumulocity-examples.git repository.
```
Comment (l2p-) [inline content/protocol-integration/lpwan-custom-codec-bundle/deploying-sample-codec-microservice.md:9]: I just saw that this repo was not migrated for unknown reasons.

Comment (BeateRixen) [reply content/protocol-integration/lpwan-custom-codec-bundle/deploying-sample-codec-microservice.md:9]: @l2p- So how to proceed then? Can you please clarify with @RobertSquire-c8y? I would actually not want to leave the links pointing to Software AG.

Comment (RobertSquire-c8y) [reply content/protocol-integration/lpwan-custom-codec-bundle/deploying-sample-codec-microservice.md:9]: that repo has not yet been transferred. The existing job using the current repo does not pass and until this is fixed I will not transfer it as I cannot know the transfer worked correctly. Waiting for this to be fixed

Comment (BeateRixen) [reply content/protocol-integration/lpwan-custom-codec-bundle/deploying-sample-codec-microservice.md:9]: Can you please let me know when this repo is fixed and transferred? I will wait with merging until then. All other repos seem to be transferred, but the links are anyway redirected, so they still work.

**Review by MWindel — APPROVED** [review]

File: content/protocol-integration/lpwan-custom-codec-bundle/deploying-sample-codec-microservice.md (hunk 1 above)
Comment (BeateRixen) [reply content/protocol-integration/lpwan-custom-codec-bundle/deploying-sample-codec-microservice.md:9]: @l2p- @RobertSquire-c8y As far as I can see the repo https://github.com/Cumulocity-IoT/cumulocity-examples has been transferred now?

Comment (RobertSquire-c8y) [reply content/protocol-integration/lpwan-custom-codec-bundle/deploying-sample-codec-microservice.md:9]: Yes that has been transferred

**Pull request merged**